bindings/python/algorithm/expose-rnea.cpp

```diff
@@ -48,13 +48,36 @@ namespace se3
               bp::return_value_policy<bp::return_by_value>());
       
 
-      bp::def("nle",
+      bp::def("nonLinearEffects",
               (const VectorXd & (*)(const Model &, Data &, const VectorXd &, const VectorXd &))&nonLinearEffects,
               bp::args("Model","Data",
                        "Configuration q (size Model::nq)",
                        "Velocity v (size Model::nv)"),
               "Compute the Non Linear Effects (coriolis, centrifugal and gravitational effects), put the result in Data and return it.",
               bp::return_value_policy<bp::return_by_value>());
+
+      bp::def("nle",
```

Comment:
By the way, if nle is a shorthand, it has to be handled like that. Which means that pinocchio.nle = pinocchio.nonLinearEffects.
As it is done now, `nle` and `nonLinearEffects`are two different functions doing the same job.